bundler.d/development.rb

```diff
@@ -1,6 +1,6 @@
 group :development do
   gem 'maruku', '~> 0.7'
-  gem 'rubocop', '0.52.1'
+  gem 'rubocop', '~> 0.53.0'
```

Comment:
hound seems to still be on 0.52.1 (https://github.com/houndci/linters/blob/master/Gemfile.lock#L78), so it won't benefit from the new rules until they upgrade their version